### PR TITLE
Fix key from nil to :index in seaborn-data

### PR DESCRIPTION
**Pull request by heronshoes**
This is a PR about #131 .

### CODE REVIEW

**Comment by heronshoes** [comment]
Also added `Datasets::SeabornData.list` to get all available dataset names.

This is useful to debug over whole datasets.

File: lib/datasets/seaborn-data.rb [hunk 1]
```diff
@@ -1,6 +1,15 @@
...
     URL_FORMAT = "https://raw.githubusercontent.com/mwaskom/seaborn-data/master/%{name}.csv".freeze
+    
+    # Show available dataset names.
+    def self.list
+      # This list is acquired by:
+      # $ ruby -rpycall -e "p PyCall.import_module('seaborn').get_dataset_names"
+      [ 'anagrams', 'anscombe', 'attention', 'brain_networks', 'car_crashes', 'diamonds',
+        'dots', 'exercise', 'flights', 'fmri', 'gammas', 'geyser', 'iris', 'mpg', 'penguins',
+        'planets', 'taxis', 'tips', 'titanic']
+    end
```
Comment (kou) [inline lib/datasets/seaborn-data.rb:12]: Could you split this feature to another pull request to merge the `nil` key support?

FYI: We should implement this feature as a dataset like `Datasets::RdatasetsList`: https://github.com/red-data-tools/red-datasets/blob/master/lib/datasets/rdatasets.rb#L5

Comment (heronshoes) [reply lib/datasets/seaborn-data.rb:12]: I should have done so. I will make it as Datasets::SeabornDataList .
Thank you for your comment.
